worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/ObservedDevice.scala

```diff
@@ -19,41 +19,41 @@ package org.apache.celeborn.service.deploy.worker.storage
 
 import java.io.File
 import java.util
-import java.util.{Set => jSet}
+import java.util.{Set => JSet}
 import java.util.concurrent.ConcurrentHashMap
 
 import scala.collection.JavaConverters._
 import scala.io.Source
 
-import org.slf4j.LoggerFactory
-
 import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.internal.Logging
 import org.apache.celeborn.common.meta.{DeviceInfo, DiskInfo, DiskStatus}
 import org.apache.celeborn.common.metrics.source.AbstractSource
+// Can Remove this if celeborn don't support scala211 in future
+import org.apache.celeborn.common.util.FunctionConverter._
 import org.apache.celeborn.common.util.JavaUtils
 
-class ObservedDevice(val deviceInfo: DeviceInfo, conf: CelebornConf, workerSource: AbstractSource) {
-
-  val logger = LoggerFactory.getLogger(classOf[ObservedDevice])
+class ObservedDevice(val deviceInfo: DeviceInfo, conf: CelebornConf, workerSource: AbstractSource)
+  extends Logging {
 
   val diskInfos = JavaUtils.newConcurrentHashMap[String, DiskInfo]()
-  deviceInfo.diskInfos.foreach { case diskInfo =>
-    diskInfos.put(diskInfo.mountPoint, diskInfo)
-  }
-  val observers: jSet[DeviceObserver] = ConcurrentHashMap.newKeySet[DeviceObserver]()
+  deviceInfo.diskInfos.foreach { diskInfo => diskInfos.put(diskInfo.mountPoint, diskInfo) }
 
-  val sysBlockDir = conf.workerDiskMonitorSysBlockDir
+  val observers: JSet[DeviceObserver] = ConcurrentHashMap.newKeySet[DeviceObserver]()
+
+  val sysBlockDir: String = conf.workerDiskMonitorSysBlockDir
   val statFile = new File(s"$sysBlockDir/${deviceInfo.name}/stat")
   val inFlightFile = new File(s"$sysBlockDir/${deviceInfo.name}/inflight")
 
-  val nonCriticalErrors = JavaUtils.newConcurrentHashMap[DiskStatus, util.Set[Long]]()
-  val notifyErrorThreshold = conf.workerDiskMonitorNotifyErrorThreshold
-  val notifyErrorExpireTimeout = conf.workerDiskMonitorNotifyErrorExpireTimeout
+  val nonCriticalErrors: ConcurrentHashMap[DiskStatus, JSet[Long]] =
+    JavaUtils.newConcurrentHashMap[DiskStatus, JSet[Long]]()
+  val notifyErrorThreshold: Int = conf.workerDiskMonitorNotifyErrorThreshold
+  val notifyErrorExpireTimeout: Long = conf.workerDiskMonitorNotifyErrorExpireTimeout
 
   var lastReadComplete: Long = -1
   var lastWriteComplete: Long = -1
-  var lastReadInflight: Long = -1
-  var lastWriteInflight: Long = -1
+  var lastReadInFlight: Long = -1
```

Comment:
nit: inflight seems to be a word
```
adjective
occurring or provided during an aircraft flight.
"inflight entertainment"
```

Reply:
we have some configurations named `*InFlight`, e.g.

```
celeborn.client.push.maxReqsInFlight
```

I think we should make them consistent, either option is fine to me.

Reply:
Alright, let's use ```*InFlight``` then.